features/other_b/series.feature

```diff
@@ -4,59 +4,53 @@ Feature: Create and Edit Series
   As a reader
   The index needs to load properly, even for authors with more than ArchiveConfig.ITEMS_PER_PAGE series
 
-  @disable_caching
-  Scenario: Three ways to add a work to a series
-    Given the following activated user exists
-      | login         | password   |
-      | author        | password   |
-      And a warning exists with name: "Choose Not To Use Archive Warnings", canonical: true
-      And the default ratings exist
-    When I am logged in as "author" with password "password"
-      And I go to the new work page
-      And I select "Not Rated" from "Rating"
-      And I check "Choose Not To Use Archive Warnings"
-      And I fill in "Fandoms" with "My Little Pony"
-      And I check "series-options-show"
-      And I fill in "work_series_attributes_title" with "Ponies"
-      And I fill in "Work Title" with "Sweetie Belle"
-      And I fill in "content" with "First little pony is all alone"
+  Scenario: Creator adds a work to a new series when the work is first posted
+    Given I am logged in as "author"
+      And I set up the draft "Sweetie Belle"
```

Comment:
This can be replaced by

I set up the draft "Sweetie Belle" as part of a series "Ponies"

And the following line removed